rulebook.tex

```diff
@@ -2523,10 +2523,10 @@ \subsubsection{Changes compared to the Main Competition}
 However, some aspects are altered to simplify the setup.
 
 \paragraph{Product Delivery}
-The delivery procedure for finished products is altered compared to the
-\ac{RCLL} rule set. In order to reduce the amount of machines required
-for participation, deliveries are made by bringing the finished product
-to the insertion area and dropping it there.
+It is possible to deliver a Product to the delivery station. To reduce the
```

Comment:
The sentence "It is possible to deliver a product to the delivery station" seems obvious.
I think you can simply leave it out and alter the next sentence sligthtly:

To reduce ..., it is possible ... drop it there, **instead of bringing it to a delivery station**.